aten/src/ATen/cuda/tunable/GemmCommon.h

```diff
@@ -153,28 +246,43 @@ struct GemmParams : OpParams {
   int64_t ldc;
 private:
   bool duplicate_inputs_;
+  bool is_reference_;
+  at::Tensor cpu_reference_;
 };
 
 template <typename T>
 struct GemmStridedBatchedParams : OpParams {
   GemmStridedBatchedParams() {
     duplicate_inputs_ = false;
+    is_reference_ = false;
   }
 
   std::string Signature() const override {
     return c10::str(transa, transb, "_", m, "_", n, "_", k, "_B_", batch);
   }
 
+  int64_t _GetCountA() const {
+    return std::min(lda, stride_a) * ((transa == 'n' || transa == 'N') ? k : m) * batch;
+  }
+
   size_t GetSizeA() const {
-    return sizeof(T) * std::min(lda, stride_a) * ((transa == 'n' || transa == 'N') ? k : m) * batch;
+    return sizeof(T) * _GetCountA();
+  }
+
+  int64_t _GetCountB() const {
+    return std::min(ldb, stride_b) * ((transb == 'n' || transb == 'N') ? n : k) * batch;
   }
 
   size_t GetSizeB() const {
-    return sizeof(T) * std::min(ldb, stride_b) * ((transb == 'n' || transb == 'N') ? n : k) * batch;
+    return sizeof(T) * _GetCountB();
+  }
+
+  int64_t _GetCountC() const {
+    return std::min(ldc, stride_c) * n * batch;
```

Comment:
Same question as before: Some of these are int64_t and others are size_t. Should they all be one or the other?